packages/synchro-charts/src/components/sc-dial/sc-dial-base/sc-dial-svg.tsx

```diff
@@ -62,13 +80,68 @@ export class ScDialSvg {
       labelSize: newSize?.labelSize || LABEL_SIZE,
       unitSize: newSize?.unitSize || UNIT_SIZE,
     };
+    this.sizeChanged = true;
```

Comment:
could we just call `positionLabelAndIcon` from here instead?

Reply:
The `positionLabelAndIcon` function is  that recalculate relative position width `label` and `icon` by the width、height and coordinate position of `label` after rendering, then re-render.

When the size is changed, the label information is obtained before the previous size change, not the latest information.